docs/griptape-cloud/knowledge-bases/accessing-data.md

```diff
@@ -0,0 +1,33 @@
+# Accessing Data in a Knowledge Base
+
+You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.
+
+## From the UI
+
+You can try out both actions from the UI on the `Test` tab of your Knowledge Base.
```

Comment:
same thing here. i dont really know what terminology we landed on. cloud console? portal?

Reply:
maybe just `From Griptape Cloud`?

Reply:
I thought I heard "cloud console", but will double check with folks on naming.